src/_ctf-private/ctf-sign-up-banner/SignupBanner.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Button, Container, Flex, Text, useBreakpointValue, useTheme } from '@chakra-ui/react';
+import Link from 'next/link';
+
+import { useSticky } from './hooks/useSticky';
+
+import CfLogo from '@icons/cf-logo.svg';
+
+export const SignupBanner = () => {
+  const isMobile = useBreakpointValue({ base: true, md: false });
+  const theme = useTheme();
+  // const MotionButton = motion(Button, { forwardMotionProps: true });
```

Comment:
Can this be removed?

Reply:
ah damn, my first time making a PR using Webstorm... the theme and color is different. Gonna clean this up.